### PR TITLE
New template

### PR DESCRIPTION
**Pull request by hyunMin11**
-Scroll 기능과 설명 기능 추가
-설명 부분에서 원하는 목록 클릭하면 해당 ppt로 이동
-ppt 움직이는 button 변경
-single layer일 경우에는 ReLU radio button 비활성화
-모든 setting이 이루어 지지 않을경우 OK버튼 비활성화
-check.html부분 design 정리

### CODE REVIEW

**Comment by hyunMin11** [comment]
리뷰부탁드립니다

File: practice/templates/practice/training/training.html [hunk 1]
```diff
@@ -1,37 +1,149 @@
...
+      <img src="{% static 'practice/images/ppt/training (2).png' %}" width="auto" height="400" alt="second image">
+      <img src="{% static 'practice/images/ppt/training (3).png' %}" width="auto" height="400" alt="third image">
+      <img src="{% static 'practice/images/ppt/training (4).png' %}" width="auto" height="400" alt="fourth image">
+      <img src="{% static 'practice/images/ppt/training (5).png' %}" width="auto" height="400" alt="first image">
+      <img src="{% static 'practice/images/ppt/training (6).png' %}" width="auto" height="400" alt="second image">
+      <img src="{% static 'practice/images/ppt/training (7).png' %}" width="auto" height="400" alt="third image">
+      <img src="{% static 'practice/images/ppt/training (8).png' %}" width="auto" height="400" alt="fourth image">
+      <img src="{% static 'practice/images/ppt/training (9).png' %}" width="auto" height="400" alt="first image">
+      <img src="{% static 'practice/images/ppt/training (10).png' %}" width="auto" height="400" alt="second image">
+      <img src="{% static 'practice/images/ppt/training (11).png' %}" width="auto" height="400" alt="third image">
```
Comment (leesr) [inline practice/templates/practice/training/training.html]: alt 값이 이미지와 맞지 않습니다.

File: practice/templates/practice/algorithm.html [hunk 2]
```diff
@@ -21,22 +21,132 @@
...
+    <ul class="w3-ul">
+      <li><h4>List</h4></li>
+      <li><span onClick="getIndex(0)">Layers</span></li>
+      <li><span onClick="getIndex(3)">Activation Function</span></li>
+      <li><span onClick="getIndex(8)">Weight Initialization</span></li>
+      <li><span onClick="getIndex(12)">Dropout</span></li>
+    </ul>
+  </div>
+  <div class="w3-container w3-cell">
+    <h3>How is the MNIST algorithm organized?</h3>
```
Comment (leesr) [inline practice/templates/practice/algorithm.html]: MNIST는 손글씨 dataset이기 때문에 "the MNIST algorithm"이라는 표현보다는 "What is Neural Network?"이나 "What is Softmax Regression?"이 적당할 것 같습니다.

File: practice/templates/practice/algorithm.html [hunk 3]
```diff
@@ -21,22 +21,132 @@
...
+      <img src="{% static 'practice/images/ppt/algorithm (5).png' %}" width="auto" height="400" alt="first image">
+      <img src="{% static 'practice/images/ppt/algorithm (6).png' %}" width="auto" height="400" alt="second image">
+      <img src="{% static 'practice/images/ppt/algorithm (7).png' %}" width="auto" height="400" alt="third image">
+      <img src="{% static 'practice/images/ppt/algorithm (8).png' %}" width="auto" height="400" alt="fourth image">
+      <img src="{% static 'practice/images/ppt/algorithm (9).png' %}" width="auto" height="400" alt="first image">
+      <img src="{% static 'practice/images/ppt/algorithm (10).png' %}" width="auto" height="400" alt="second image">
+      <img src="{% static 'practice/images/ppt/algorithm (11).png' %}" width="auto" height="400" alt="third image">
+      <img src="{% static 'practice/images/ppt/algorithm (12).png' %}" width="auto" height="400" alt="fourth image">
+      <img src="{% static 'practice/images/ppt/algorithm (13).png' %}" width="auto" height="400" alt="first image">
+      <img src="{% static 'practice/images/ppt/algorithm (14).png' %}" width="auto" height="400" alt="second image">
```
Comment (leesr) [inline practice/templates/practice/algorithm.html]: alt 값이 그림과 맞지 않습니다.
